pkgs/misc/themes/arc/default.nix

```diff
@@ -1,43 +1,55 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
+{ stdenv, fetchFromGitHub, sassc, autoreconfHook, pkgconfig, gnome3
+, gnome-themes-standard, gtk-engine-murrine, optipng, inkscape}:
 
 let
-  # treat versions newer than 3.22 as 3.22
-  gnomeVersion = if stdenv.lib.versionOlder "3.22" gnome3.version then "3.22" else gnome3.version;
+  # per README: For GNOME 3.24 and 3.26, use --with-gnome=3.22
+  gnomeVersion = if gnome3.version == "3.24" || gnome3.version == "3.26" then "3.22" else gnome3.version;
   pname = "arc-theme";
 in
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
-  version = "2017-05-12";
+  version = "2018-07-13";
```

Comment:
It may be better to use a released version, instead of a simple revision. The latest released version is `20180715`, only two days after the revision used in the PR.